src/main/java/racingcar/domain/Car.java

```diff
@@ -0,0 +1,45 @@
+package racingcar.domain;
+
+import racingcar.strategy.MovingStrategy;
+
+import java.util.Objects;
+
+public class Car {
+    private static final int INIT_POSITION = 0;
+    private int position;
```

Comment:
```suggestion
    private static final int INIT_POSITION = 0;

    private int position;
```

상수와 인스턴스 변수 사이에는 한 줄을 띄워주세요~